### DISCUSSION TITLE
Auto generate number

### DISCUSSION THREAD

**Discussion by MonsterITServices** [YetiForceCompany/YetiForceCRM#16713] [General]
Hi,

Is it possible to auto generate a number so if a user creates a record it starts like S0012023?  The last bit is the year and the first bit is what we define it? Is this possible or how much would it cost to build this?

**Comment by mariuszkrzaczkowski** [comment]
It's already in the system, see this panel:
https://gitdeveloper.yetiforce.com/index.php?module=RecordNumbering&parent=Settings&view=CustomRecordNumbering

**Reply by mariuszkrzaczkowski** [reply]
![image](https://user-images.githubusercontent.com/10198654/205899461-aa0673c3-e011-4740-886e-67b68d85cfe3.png)
